packages/connect-examples/expo-playground/app/services/previewHardwareParams.ts

```
@@ -0,0 +1,74 @@
+
+    // 通用提取：coin_name / chain_id / coin
+    if (p.coin_name || p.coin) preview.coin_name = p.coin_name ?? p.coin;
+    if (p.chain_id !== undefined) preview.chain_id = p.chain_id;
+
+    // 通用：脚本类型与关闭标志（兼容大小写）
+    if (p.script_type !== undefined) {
+      preview.script_type = p.script_type;
+    } else if (Array.isArray(preview.address_n)) {
+      try {
+        preview.script_type = getScriptType(preview.address_n as number[]);
+      } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)